packages/flutter_tools/test/general.shard/windows/visual_studio_test.dart

```diff
@@ -766,7 +766,7 @@ void main() {
       expect(fixture.logger.warningText, contains('Warning: Unexpected vswhere.exe JSON output'));
     });
 
-    testWithoutContext('Everything returns good values when VS is present with all components', () {
+    testUsingContext('Everything returns good values when VS is present with all components', () {
```

Comment:
we should be overriding whatever global these tests are depending on

Reply:
Added `OperatingSystemUtils` to `VisualStudio` class, and reverted back to using `testWithoutContext`.